e2e/test/csi_driver_test.go

```diff
@@ -61,33 +61,71 @@ var _ = Describe("Linode CSI Driver", func() {
 
 	Describe("Test", func() {
 		Context("Simple", func() {
+			var (
+				f    *framework.Invocation
+				size string
+				// storageClass = "linode-block-storage"
+				sts          *appsv1.StatefulSet
+				pod          *core.Pod
+				pvc          *core.PersistentVolumeClaim
+				file         = "/data/file.txt"
+				storageClass = "linode-block-storage"
+
+				writeFile = func(filename string) {
```

Comment:
It's a bit odd to duplicate this here (same for `readFile`).

Reply:
ya totally agree, thinking about coalescing too! ill do this after i get them running